src/library_html5.js

```diff
@@ -362,13 +362,9 @@ var LibraryHTML5 = {
     return GL.offscreenCanvases[target.substr(1)] // Remove '#' prefix
     // If not found, if one is querying by using DOM tag name selector 'canvas', grab the first
     // OffscreenCanvas that we can find.
-     || (target == 'canvas' && Object.keys(GL.offscreenCanvases)[0])
-    // If that is not found either, query via the regular DOM selector.
-#if PTHREADS
-     || (typeof document != 'undefined' && document.querySelector(target));
-#else
-     || document.querySelector(target);
-#endif
+     || (target == 'canvas' && Object.values(GL.offscreenCanvases)[0])
```

Comment:
Is this also missing the `.canvas` similar to in #22958

Reply:
I was worried it would, but in fact we want the whole offscreenCanvases entry because the caller might need the shared canvas pointer.   All of the callers do a check for the presence of an .offscreenCanvas field.